testutil/integration/grpc/feemarket.go

```diff
@@ -0,0 +1,15 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package grpc
+
+import (
+	"context"
+
+	feemarkettypes "github.com/evmos/evmos/v14/x/feemarket/types"
+)
+
+// GetBaseFee returns the base fee from the feemarket module.
+func (gqh *IntegrationHandler) GetBaseFee() (*feemarkettypes.QueryBaseFeeResponse, error) {
+	feeMarketClient := gqh.network.GetFeeMarketClient()
+	return feeMarketClient.BaseFee(context.Background(), &feemarkettypes.QueryBaseFeeRequest{})
```

Comment:
why are we using an empty context here, and not `gph.network.GetContext()`?

Reply:
This query simulates a user that is making a query from an external agent that has no access to the context.. the network conext is requested internally from the GetFeeMarketClient to simulate that grpc client